doc/operations/data_persistence.rst

```diff
@@ -0,0 +1,26 @@
+Data Persistence
```

Comment:
I think it would be good to move the "Best Practices" section from [here](https://microsoft.github.io/CCF/main/operations/ledger_snapshot.html#best-practices) to this new page.